### ISSUE TITLE
pre-trained ML/CNN model 

### ISSUE THREAD

**Issue by cluhaowie** [ryanlayer/samplot#156]
Would it be possible to create some sort of picture database with label(e.g., SV type, genotype, etc.) so people can just train the model with the samplot output? Maybe a Kaggle competition for a model selection?

**Comment by mchowdh200** [comment]
- as far as creating something akin to a picture database, [SVPlaudit](https://github.com/jbelyeu/SV-plaudit) is a curation tool that was built upon samplot visualizations to enable a fast manual curation interface for those wanting to create high confidence sets of SVs.
- this would be a good way to generate more data for the purpose of training new models or fine-tuning old ones.
- at this time the infrastructure for fine tuning a samplot-ml model hasn't been built out yet, but its something we plan on doing in the near future (of course since it was built with tensorflow, one could load the model and implement their own fine-tuning training loop with data of their own creation).
